caiman/gui/gui_pyqtgraph_layout.py

```diff
@@ -1,15 +1,15 @@
+import cv2
+import numpy as np
 import pyqtgraph as pg
+import scipy
 from pyqtgraph import FileDialog
-from pyqtgraph.Qt import QtCore, QtGui
-from pyqtgraph.parametertree import Parameter, ParameterTree, ParameterItem, registerParameterType
-import numpy as np
+from pyqtgraph.Qt import QtGui
+from pyqtgraph.parametertree import Parameter, ParameterTree
+
 import caiman as cm
 from caiman.source_extraction.cnmf.cnmf import load_CNMF
-from caiman.source_extraction.cnmf.online_cnmf import load_OnlineCNMF
+from caiman.source_extraction.cnmf.params import CNMFParams
 
-import cv2
-import scipy
-import os
```

Comment:
I ran PyCharm's import optimizer, which reordered the imports a bit and removed unused imports